cypress/utils/utils.ts

```diff
@@ -140,16 +140,16 @@ export function getTableColumnData(columnName: string): Array<string> {
     return itemList;
 }
 
-export function verifySortAsc(listToVerify: Array<any>): void {
+export function verifySortAsc(listToVerify: Array<any>, unsortedList: Array<any>): void {
```

Comment:
@sshveta  I did this approach before, but avoided it, as this _sortBy() function does not modify the original array. So, to optimize the whole solution, getting an unsorted list before performing any kind of sort, was avoided. 
Because, anyways, at last we need to sort or reverse sort the captured array from UI. So, if it is already sorted and then we re-sort it, it just optimizes the code, unless, we are disturbing the original array.

Reference - [Cypress sorting a table](https://www.cypress.io/blog/2020/07/27/sorting-the-table/)